components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/GatewayCmdUtils.java

```diff
@@ -287,6 +296,16 @@ public static void createProjectStructure(String projectName) throws IOException
         String definitionsPath = projectDir + File.separator + GatewayCliConstants.PROJECT_API_DEFINITIONS_DIR;
         createFolderIfNotExist(definitionsPath);
 
+        String grpcDefinitionsPath = projectDir + File.separator + GatewayCliConstants.PROJECT_GRPC_DEFINITIONS_DIR;
+        String grpcCustomOptionDirPath = grpcDefinitionsPath + File.separator +
+                GatewayCliConstants.PROJECT_GRPC_OPTIONS_DIR;
+        String grpcCustomOptionsFilePath = grpcCustomOptionDirPath + File.separator +
+                GatewayCliConstants.PROJECT_GRPC_OPTIONS_FILE;
+        createFolderIfNotExist(grpcDefinitionsPath);
+        createFolderIfNotExist(grpcCustomOptionDirPath);
+        FileUtils.copyFile(new File(getGrpcFolderLocation() + "/" + GatewayCliConstants.PROJECT_GRPC_OPTIONS_FILE),
```

Comment:
Is there a specific reason we can not use File.separator here

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/757/commits/9aa37eddca472d78c58f979901c46c3ae69931e4.